src/main/java/com/google/devtools/build/lib/query2/cquery/ConfiguredTargetQueryEnvironment.java

```diff
@@ -252,6 +252,13 @@ private static BuildConfiguration mergeEqualBuildConfiguration(
             OutputType.JSON),
         new BuildOutputFormatterCallback(
             eventHandler, cqueryOptions, out, skyframeExecutor, accessor),
+        new GraphOutputFormatterCallback(
```

Comment:
Don't forget to update the documentation. I think you'll want to remove the bullet point at https://github.com/bazelbuild/bazel/blob/74bd30243515055f41c1e6cf3fc2ce87ae644138/site/docs/cquery.html#L569 and then also add a section to `#output-formats` on the new support for `--output=graph`.

Reply:
Done.